src/components/Icon/ico-payments.vue

```diff
@@ -0,0 +1,15 @@
+<template>
+  <svg
+    class="eds-icon"
+    height="24"
+    viewBox="0 0 24 24"
+    width="24"
+    xmlns="http://www.w3.org/2000/svg"
+  >
+    <path
```

Comment:
It looks like orders/payments are the same icons

<img width="305" alt="Screenshot 2024-05-27 at 16 32 56" src="https://github.com/everli/design-system-frontend/assets/2059481/3ce80c38-cb29-43bb-8f9f-9aeb909d5574">

Reply:
also, please update branch with dev because I see a lot of conflicted files listed below